language-bindings/python/README.md

```diff
@@ -1,31 +1,34 @@
 # wamr-python
 
-## Installation
+The WAMR Python package contains a set of high-level bindings for WAMR API and WASM-C-API.
 
-### Installing from the source code
+## Installation
 
-Installing from local source tree is in _development mode_. The package appears to be installed but still is editable from the source tree.
+To Install from local source tree in _development mode_ run the following command,
 
 ```bash
-$ python -m pip install -e /path/to/wamr-root/binding/python
+python -m pip install -e .
```

Comment:
this doesn't run create_lib.sh for me. (otoh, `pip install .` does.)
is it expected?

Reply:
If it doesn't run it, something has to be fixed

Reply:
@yamt you uploaded PR #1978 to fix it, right?
@tonibofarull Could you help review #1978 and #1977? Thanks.

Reply:
> @yamt you uploaded PR #1978 to fix it, right?

no. the PR is about a separate topic.
i don't have enough setup.py knowledge to fix this right now.

Reply:
To install it we need to run:
```
python language-bindings/python/setup.py install --user
```
instead of pip install.